### PR TITLE
Rename the default gateway environment "Production and Sandbox" to "Default"

### PR DESCRIPTION
**Pull request by rmsamitha**
Fixes https://github.com/wso2/product-apim/issues/10477
Related PR for product-apim is https://github.com/wso2/product-apim/pull/10906

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/java/org/wso2/carbon/apimgt/rest/api/common/RestApiConstants.java [hunk 1]
```diff
@@ -294,7 +294,7 @@
     public static final String ERROR_TOKEN_EXPIRED = "Access token is expired";
     public static final String ERROR_SCOPE_VALIDATION_FAILED = "You cannot access API as scope validation failed";
 
-    public static final String DEFAULT_ENVIRONMENT = "Production and Sandbox";
+    public static final String DEFAULT_ENVIRONMENT = "Default";
```
[the remark this answers is not in the document]
Comment (rmsamitha) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/java/org/wso2/carbon/apimgt/rest/api/common/RestApiConstants.java]: Fixed with 6fc2e54

File: components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java [hunk 2]
```diff
@@ -132,7 +132,7 @@ public static GenericArtifact getSampleAPIArtifactForTenant() throws GovernanceE
         artifact.setAttribute("overview_apiSecurity","oauth2,oauth_basic_auth_api_key_mandatory");
         artifact.setAttribute("URITemplate_authType0","Application & Application User");
         artifact.setAttribute("overview_keyManagers","[\"all\"]");
-        artifact.setAttribute("overview_environments","Production and Sandbox");
+        artifact.setAttribute("overview_environments","Default");
```
Comment (tharindu1st) [inline components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java:135]: remove this property itself.

File: components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java [hunk 3]
```diff
@@ -215,7 +215,7 @@ public static GenericArtifact getSampleAPIProductArtifact() throws GovernanceExc
         artifact.setAttribute("overview_visibleTenants",null);
         artifact.setAttribute("overview_name","APIProductTest");
         artifact.setAttribute("overview_apiSecurity","oauth2,oauth_basic_auth_api_key_mandatory,basic_auth,api_key");
-        artifact.setAttribute("overview_environments","Production and Sandbox");
+        artifact.setAttribute("overview_environments","Default");
```
Comment (tharindu1st) [inline components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java:218]: this property as well

**Review by tharindu1st — CHANGES_REQUESTED** [review]

File: components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java (hunk 2 above)
Comment (rmsamitha) [reply components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java:135]: The underlying method and class is used only in test cases. So this attribute cannot be removed.

File: components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java (hunk 3 above)
Comment (rmsamitha) [reply components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java:218]: The underlying method and class is used only in test cases. So this attribute cannot be removed.

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**